src/components/Attachments/AttachmentCarousel/index.native.tsx

```diff
@@ -38,23 +40,28 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
             targetAttachments = extractAttachments(CONST.ATTACHMENT_TYPE.REPORT, {parentReportAction, reportActions});
         }
 
-        const initialPage = targetAttachments.findIndex(compareImage);
+        let attachmentIndex = targetAttachments.findIndex(compareImage);
+        const prevAttachmentIndex = attachments.findIndex(compareImage);
```

Comment:
NAB. Another naming clarity. `targetAttachments` is confusing.

- targetAttachments -> newAttachments
- attachmentIndex -> newIndex
- prevAttachmentIndex -> index

Reply:
done